src/actions/userActions.js

```diff
@@ -0,0 +1,24 @@
+import { userServices } from "services/userServices";
+import { userActionTypesConstants } from "constants/userConstants";
+
+const { USER_REQUEST_SUCCESS } = userActionTypesConstants;
+
+const login = (email, password) => {
+  return async (dispatch) => {
+    try {
```

Comment:
Add dispatch for `USER_REQUEST` here (There's another comment explaining this better in other file)